developer/environment.rst

```diff
@@ -22,7 +22,7 @@ To update environment variables:
 Package-level
 ~~~~~~~~~~~~~
 
-For each built package ``colcon`` generates a set of package-level scripts (one for each supported shell type): ``share/<package_name>/package.<ext>``.
+For each built package ``colcon`` generates a set of package-level scripts (one for each supported shell type): ``<package_name>/share/<package_name>/package.<ext>``.
```

Comment:
This isn't entirely correct. Maybe instead it should be clarified that the previously mentioned path is relative to the `install prefix` of a package.

In the default case the install prefix points to `install/<package_name>` but when using `--merge-install` it is just `install`.

Same below.

Reply:
Ah, indeed, thanks for the clarification!

> Maybe instead it should be clarified that the previously mentioned path is relative to the install prefix of a package.

:+1: